modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,556 @@
+
+Moreover, in the circumstances where the `autocompleter` does not activate, even typing the entire Merge tag does not result in the string being turned in to a Merge tag.
+
+There is a workaround, however.
+
+If typing the Merge tags prefix does not work as expected, clicking the Merge tags toolbar button and selecting the desired Merge tag from the resultant menu inserts a functioning Merge tag.
+
+=== Merge tags does not validate inserted strings that match merge tag syntax
+
+If a string is pasted or otherwise inserted into a {productname} document and that string presents as a complete merge tag, the Merge tag plugin will turn the string into an apparently working merge tag, regardless of the string’s content.
+
+For example, if a {productname} instance is running the Merge tags plugin and the plugin is using the default prefix and suffix characters — `+{{+` and `+}}+` — pasting
+
+`{{not.a.merge.tag.value.for.this.document}}`
+
+into the document results in the pasted string being turned in to an apparently working tag, ready to take whatever values `not.a.merge.tag.value.for.this.document` is the placeholder for.
+
+There is no current workaround.
+
+=== A Merge tags UI element has not been translated
+
+{productname} 6.2 adds a new UI option to menu toolbar buttons. These buttons can now present a search field at the top of the menu.
+
+As text is entered in to this search field, the menu contents update in real-time. And, when nothing in the menu matches the entered string, the MenuButton presents a _No matches_ result.
+
+This particular string, _No matches_ has not been translated.
+
+There is no current workaround. This error is slated for fixing in a future {productname} release.
+
+=== The Autocorrect plugin over-rides auto-capitalization.
+
+As a consequence of {pluginname} being case-sensitive, if
+
+* both `autocorrect` and `capitalization` are active; and
+* a word from the specified spelling error set is mis-typed at the beginning of a sentence;
+
+the spelling is corrected but the initial letter is not capitalized.
+
+=== The Autocorrect plugin’s auto-capitalization does not work on text immediately following a soft return in non-empty {productname} documents.
+
+Auto-capitalization will not operate on the entered string if
+
+* a {productname} document contains any data other than a soft return character (Unicode U+2028, commonly entered by typing *Shift+Return*); and
+
+* the character immediately preceding an entered string is a soft return character.
+
+=== The Autocorrect plugin’s auto-capitalization interferes with the following Premium plugins
+
+* `+mediaembed+`, the xref:introduction-to-mediaembed.adoc[Enhanced Media Embed] Premium plugin.
+
+* `linkchecker`, the xref:linkchecker.adoc[Link Checker] Premium plugin.
+
+=== When the Spell Checker Pro plugin is used, Autocorrect’s autocorrection is disabled when a non-English language is selected.
```

Comment:
```suggestion
=== When the Spell Checker Pro plugin is used, Spelling Autocorrect's autocorrect and capitalization are disabled when a non-English language is selected.
```

Reply:
We're a bit inconsistent with calling it `autocorrect` vs `autocorrection` unfortunately.